### ISSUE TITLE
Error when using the letters Ä Ö Å

### ISSUE THREAD

**Issue by mightymiles** [Nuitka/Nuitka#377]
Hi
Im getting errors when i try to compile my code who contains Ä Ö and  Å letters when i remove them the code compiles just fine. Is there a work around this or is it a bug?

Here is the Error Report i get.

```
c:\test>python -m nuitka --follow-imports test4.py
cl /Fotest4.build\module.__main__.obj /c "test4.build\module.__main__.cpp" /TP /
nologo /EHsc /J /Gd /Ox /MT /D_NUITKA_SYSFLAG_BYTES_WARNING=0 /D_NUITKA_SYSFLAG_
NO_SITE=0 /D_NUITKA_SYSFLAG_VERBOSE=0 /D_NUITKA_SYSFLAG_UTF8=0 /D_NUITKA_SYSFLAG
_OPTIMIZE=0 /D__NUITKA_NO_ASSERT__ /D_NUITKA_CONSTANTS_FROM_RESOURCE /D_NUITKA_F
ROZEN=0 /D_NUITKA_MODULE_COUNT=2 /D_NUITKA_EXE /IC:\Users\xzenon\AppData\Local\P
rograms\Python\Python37-32\include /IC:\Users\xzenon\AppData\Local\Programs\Pyth
on\Python37-32\PC /Itest4.build /IC:\Users\xzenon\AppData\Local\Programs\Python\
Python37-32\lib\site-packages\nuitka\build\include /IC:\Users\xzenon\AppData\Loc
al\Programs\Python\Python37-32\lib\site-packages\nuitka\build\static_src
test4.build\module.__main__.cpp(323): error C3872: '0x24': this character is not
 allowed in an identifier
test4.build\module.__main__.cpp(784): error C3872: '0x24': this character is not
 allowed in an identifier
test4.build\module.__main__.cpp(3112): error C3872: '0x24': this character is no
t allowed in an identifier
test4.build\module.__main__.cpp(3115): error C3872: '0x24': this character is no
t allowed in an identifier
test4.build\module.__main__.cpp(4504): error C3872: '0x24': this character is no
t allowed in an identifierscons: *** [test4.build\module.__main__.obj] Error 2
```

**Comment by kayhayen** [comment]
There is a fix on factory. Seems unicode will work, but Latin1 did not. Now Nuitka restrains the identifiers to "ascii", so a function 'bäh' and variable reference böl are working for me.

Checkout http://nuitka.net/doc/factory.html or just wait until this becomes released.

Yours,
Kay

**Comment by mightymiles** [comment]
Thank you it works.

**Comment by matkuki** [comment]
Hi @kayhayen 
I have source code that uses characters '__čšž__', which works fine in CPython, but Nuitka throws:
```
Traceback (most recent call last):
  File "C:\Python37\lib\site-packages\nuitka-0.6.4rc3-py3.7.egg\nuitka\__main__.py", line 184, in <module>
    main()
  File "C:\Python37\lib\site-packages\nuitka-0.6.4rc3-py3.7.egg\nuitka\__main__.py", line 177, in main
    MainControl.main()
  File "C:\Python37\lib\site-packages\nuitka-0.6.4rc3-py3.7.egg\nuitka\MainControl.py", line 750, in main
    result, options = compileTree(main_module=main_module)
  File "C:\Python37\lib\site-packages\nuitka-0.6.4rc3-py3.7.egg\nuitka\MainControl.py", line 643, in compileTree
    makeSourceDirectory(main_module=main_module)
  File "C:\Python37\lib\site-packages\nuitka-0.6.4rc3-py3.7.egg\nuitka\MainControl.py", line 398, in makeSourceDirectory
    writeSourceCode(filename=c_filename, source_code=source_code)
  File "C:\Python37\lib\site-packages\nuitka-0.6.4rc3-py3.7.egg\nuitka\MainControl.py", line 577, in writeSourceCode
    output_file.write(source_code.encode("latin1"))
UnicodeEncodeError: 'latin-1' codec can't encode character '\u010d' in position 437485: ordinal not in range(256)
```
is this relevant to this issue?

I'm on Windows 10, Python 3.7, Nuitka on factory branch

Matic

**Comment by kayhayen** [comment]
Now on develop

**Comment by kayhayen** [comment]
Released as 0.6.4
